### PR TITLE
feat(browser-support): show option to join via browser on supported mobile browsers

### PR DESCRIPTION
**Pull request by jallamsetty1**
Currently Chrome, Edgium and Firefox are supported on Android
Only Safari is supported on iOS

### CODE REVIEW

File: react/features/base/environment/environment.js [hunk 1]
```diff
@@ -81,6 +83,18 @@ export function isSupportedBrowser() {
...
 }
 
+/**
+ * Returns whether or not the current environment is a supported
+ * browser on a mobile device.
+ *
+ * @returns {boolean}
+ */
+export function isSupportedMobileBrowser() {
+    return (Platform.OS === 'android' && browser.isChrome())
```
Comment (saghul) [inline react/features/base/environment/environment.js:93]: maybe use isChromiumBased? There are a number of Chromium derivatives already such as Bromite https://www.bromite.org/

File: react/features/base/environment/environment.js [hunk 2]
```diff
@@ -81,6 +83,18 @@ export function isSupportedBrowser() {
...
+        || (Platform.OS === 'android' && browser.isFirefox())
+        || (Platform.OS === 'ios' && browser.isSafari());
```
Comment (saghul) [inline react/features/base/environment/environment.js:95]: Do we know if web views are detected as Safari. IIRC WebRTC does not work in them.

File: react/features/deep-linking/components/DeepLinkingMobilePage.web.js [hunk 3]
```diff
@@ -139,6 +148,15 @@ class DeepLinkingMobilePage extends Component<Props> {
                         { t(`${_TNS}.openApp`) }
                         {/* </button> */}
                     </a>
+                    {
+                        isSupportedMobileBrowser()
+                            && <a
```
Comment (saghul) [inline react/features/deep-linking/components/DeepLinkingMobilePage.web.js:153]: addd target blank for external API users?

File: react/features/base/environment/environment.js (hunk 1 above)
Comment (jallamsetty1) [reply react/features/base/environment/environment.js:93]: isChrome check would cover all these chromium based browsers, the only exception being Opera. Is it safe to just enable it for all those browsers w/o testing them ?

File: react/features/deep-linking/components/DeepLinkingMobilePage.web.js (hunk 3 above)
Comment (jallamsetty1) [reply react/features/deep-linking/components/DeepLinkingMobilePage.web.js:153]: my bad, how could I forget that :)

File: react/features/base/environment/environment.js (hunk 2 above)
Comment (jallamsetty1) [reply react/features/base/environment/environment.js:95]: I don't know, I would like to incorporate that check in browser-detection in js-utils which is what I will be working on next.

**Comment by jallamsetty1** [comment]
Thanks for the review @saghul, I will merge https://github.com/jitsi/jitsi-meet/pull/6717 and will then update this.

**Comment by jallamsetty1** [comment]
@saghul, I have made the changes you suggested, can you PTAL, thanks !

**Comment by saghul** [comment]
Nice! I can test the other browsers once it’s deployed in alpha.

**Pull request merged**